cmd/notation/cert.go

```diff
@@ -312,7 +315,7 @@ func listCerts(opts *certListOpts) error {
 			}
 		}
 
-		paths = dir.Path.ConfigFS.ListAllPath(dir.TrustStoreDir, "x509", "tsa", namedStore)
+		paths = dir.Path.ConfigFS.ListAllPath(dir.TrustStoreDir, "x509", "signingAuthority", namedStore)
```

Comment:
Why?
According to spec,
```
Timestamping Certificates: The x509/tsa trust store contains named stores with Time Stamping Authority (TSA) root certificates. NOTE TSA based timestamping will not be available in notation RC1.
```

Reply:
We will have 3 types, namely ca, tsa, and signingAuthority. We will support tsa after RC.1.